doozer/doozerlib/cli/release_gen_payload.py

```diff
@@ -236,7 +237,7 @@ def default_imagestream_base_name(version: str, runtime: Runtime) -> str:
 
 
 def default_imagestream_base_name_generic(version: str, build_system) -> str:
```

Comment:
This needs to happen for named assemblies too, pointing to `assembly_imagestream_base_name_generic()`

Reply:
Thanks - updated